api/v1beta2/foundationdb_database_configuration.go

```diff
@@ -76,9 +77,11 @@ type Region struct {
 // ExcludedServers represents the excluded servers in the database configuration
 type ExcludedServers struct {
 	// The Address of the excluded server.
+	// +kubebuilder:validation:MaxLength=20
```

Comment:
```suggestion
	// +kubebuilder:validation:MaxLength=48
```

The length. `20` is not enough for IPv6 e.g. `[2001:0db8:85a3:0000:0000:8a2e:0370:7334]:65400'`